bin/tsc.js

```diff
@@ -21,6 +21,12 @@ var ts;
         ExitStatus[ExitStatus["DiagnosticsPresent_OutputsGenerated"] = 2] = "DiagnosticsPresent_OutputsGenerated";

```

Comment:
can you please remove the bin folder from this PR. we do not do that in branches.